### DISCUSSION TITLE
Add sorting of forks

### DISCUSSION THREAD

**Discussion by jac0-b** [community/community#4545] [Other Feature Feedback, Questions, & Ideas]
Allow sorting of repos on the forks page. Here are some suggested options that could be included:

- Number of stars ⭐
- Alphabetical (current) 
- Time since most recent commit
- Number of commits (or even more specific like number of commits ahead of forked repo)
- Number of contributors
- Number of forks

If you like this idea **upvote it ⬆ instead of giving it a thumbs up**.

**Comment by dayyass** [comment]
It would be also great, if forks, that differs from base repository for at least one commit, were highlighted.

**Reply by jxu** [reply]
also mentioned in #5185

**Comment by jxu** [comment]
Agree with this. The most important use is to find active forks of unmaintained projects.

**Reply by endolith** [reply]
https://addons.mozilla.org/en-US/firefox/addon/lovely-forks/ helps a little
